app/controllers/users/piv_cac_authentication_setup_controller.rb

```diff
@@ -28,6 +26,11 @@ def delete
       redirect_to account_url
     end
 
+    def redirect_to_piv_cac_service
+      create_piv_cac_nonce
+      redirect_to PivCacService.piv_cac_service_link(piv_cac_nonce)
+    end
```

Comment:
Are there any security concerns with this method being executable via a GET request without needing to be authenticated?

Reply:
maybe -- it might be better if we required *some* kind of authentication -- either that we're at the point we want a second factor, or we're ready to set up a second factor. I'll add that. Have to do a rebase anyway.

Reply:
Rebasing brought in a `before_action :authenticate_user!` line. Manual testing seems to work -- without logging in, I'm put back on the login page, and I can add a piv/cac during account creation.